### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by rnzenha-s**
## Proposed changes

Update changelog to include latest images and latest release.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [X] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [X] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: docs/user_guide/5_changelog.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+* Fixing bug config file pid parameters being erased when saving selected
+* Rounding up values for joint slider
+
+### Version 1.0.16
+* Fixed linter errors (1)
+* Improve realtime publisher fast pid divisor
+* Fix joint position/velocity filter
+* Fixed linter errors (2)
+* Fix broken rosbags
+* Increase timeout time
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: Increase timeout (for what?)

File: docs/user_guide/5_changelog.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+### Version 1.0.17
+* Update repository.rosinstall
+* Fix handling of active rosbags
+* Remove default hand serial parameter
+* Remove sr_ethercat_config
+* Fixing bug config file pid parameters being erased when saving selected
+* Rounding up values for joint slider
+
+### Version 1.0.16
+* Fixed linter errors (1)
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: If you have multiple occurrences of "Fixed linter errors" you can just squash them into one.

File: docs/user_guide/5_changelog.md [hunk 3]
```diff
@@ -2,6 +2,31 @@
 
 
 ## ROS Noetic
+
+### Version 1.0.18 (current noetic-release)
+* Fixing demo behaviour when tactile sensors are installed (1)
+* Fixing demo behaviour when tactile sensors are installed (2)
+
+### Version 1.0.17
+* Update repository.rosinstall
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: In what repo, what's the update (My guess, it's the dependency on sr_ethercat_config (sr_config?))

File: docs/user_guide/5_changelog.md [hunk 4]
```diff
@@ -2,6 +2,31 @@
 
 
 ## ROS Noetic
+
+### Version 1.0.18 (current noetic-release)
+* Fixing demo behaviour when tactile sensors are installed (1)
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: Squash into one

File: docs/user_guide/5_changelog.md [hunk 5]
```diff
@@ -2,6 +2,31 @@
...
+
+### Version 1.0.16
+* Fixed linter errors (1)
+* Improve realtime publisher fast pid divisor
+* Fix joint position/velocity filter
+* Fixed linter errors (2)
+* Fix broken rosbags
+* Increase timeout time
+* Fixing arm only launch
+* Fixed the linter errors
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: Same meaning as "Fixed linter errors" above

File: docs/user_guide/5_changelog.md [hunk 6]
```diff
@@ -2,6 +2,31 @@
...
+* System Health Node
```
Comment (maxzieba) [inline docs/user_guide/5_changelog.md]: Added new functionality SystemHealthNode?

**Review by maxzieba — CHANGES_REQUESTED** [review]

File: docs/user_guide/5_changelog.md (hunk 6 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Improved

File: docs/user_guide/5_changelog.md (hunk 5 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: improved

**Review by maxzieba — APPROVED** [review]

File: docs/user_guide/5_changelog.md (hunk 4 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Done

File: docs/user_guide/5_changelog.md (hunk 3 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Improved

File: docs/user_guide/5_changelog.md (hunk 2 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Done

File: docs/user_guide/5_changelog.md (hunk 1 above)
Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Improved

File: docs/user_guide/5_changelog.md [hunk 7]
```diff
@@ -95,7 +116,7 @@
 * Showing allowed options for general info template
 * Re-write data visualizer
 
-### Version 1.0.0 (previous noetic-release)
+### Version 1.0.0
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md]: It will be good to keep this comment for previous releases.

Comment (maxzieba) [reply docs/user_guide/5_changelog.md]: should this comment be to every version that was used for release or only the 1 before the newest release?

Comment (beatrizleon) [reply docs/user_guide/5_changelog.md]: It should be there for all of them

Comment (rnzenha-s) [reply docs/user_guide/5_changelog.md]: Added previous noetic-release information now

**Review by jeferal — APPROVED** [review]

**Review by beatrizleon — APPROVED** [review]

**Review by biofotis — APPROVED** [review]

**Pull request merged**